### ISSUE TITLE
VSCode doesn't configure .vsix files on MacOS Monterey 12.x

### ISSUE THREAD

**Issue by SpencerGreene** [wpilibsuite/WPILibInstaller-Avalonia#262]
When doing a fresh install of wpilib on a clean Mac running 12.x, the VSCode extensions are not configured. They are present in the ~/wpilib/2022/vsCodeExtensions folder, but they are not active in VSCode.  To activate them the user has to manually go to the extensions tab in VSCode, "Install from VSIX" for each extension, then restart VSCode.

Expected behavior: immediately after the install and without any further steps, the wpilib icon is visible at top right of VSCode. This does work correctly when installing on MacOS 10.15 and on Windows 10.

Observed behavior: when installing on MacOS 12.x, open VSCode as installed by wpilib, the wpilib icon is not visible and the command palette does not include wpilib extensions. After workaround of manually installing five .VSIX then all works as expected.

Here are the configurations I have tested:
. Confirmed that it works correctly on Windows 10 (Intel) and MacOS 10.15 (Intel)
. Confirmed that it fails on MacOS 12.3.1 (Intel) and MacOS 12.4 (Intel), multiple machines.

See also: https://www.chiefdelphi.com/t/wpilib-simulation-not-working-for-me-on-macos-12-3/412409

**Comment by sciencewhiz** [comment]
On a fresh install, what happens if you run the following in the terminal (repeat for the rest of the vsix). This is what the installer calls to install the extensions, and I'm hoping there's something in the terminal saying why the extensions failed to install.
`~/wpilib/2022/vscode/Visual\ Studio\ Code.app/Contents/Resources/app/bin/code --install-extension ~/wpilib/2022/vsCodeExtensions/WPILib.vsix`

**Comment by prateekma** [comment]
```
/Users/prateekma/wpilib/2022/vscode/Visual Studio Code.app/Contents/Resources/app/bin/code: line 6: python: command not found
/Users/prateekma/wpilib/2022/vscode/Visual Studio Code.app/Contents/Resources/app/bin/code: line 10: ./MacOS/Electron: No such file or directory
```

**Comment by sciencewhiz** [comment]
This should be fixed in VS Code 1.65. https://github.com/microsoft/vscode/issues/134635

**Comment by sciencewhiz** [comment]
You could try installing the version here, which uses VS Code 1.66.2. https://github.com/wpilibsuite/WPILibInstaller-Avalonia/actions/runs/2243725711

**Comment by SpencerGreene** [comment]
@sciencewhiz I tried that build, specif the .zip file at https://github.com/wpilibsuite/WPILibInstaller-Avalonia/suites/6310182351/artifacts/226451217. It expands to .dmg but then the 'verifying artifacts file' step fails with "System.Exception: The artifacts file was damaged."

**Comment by SpencerGreene** [comment]
I'm not sure if there are any logs it stores that I should copy?

**Comment by SpencerGreene** [comment]
> You could try installing the version here, which uses VS Code 1.66.2. https://github.com/wpilibsuite/WPILibInstaller-Avalonia/actions/runs/2243725711

Yes!  This did work for me on 12.4 (Intel).

**Comment by sciencewhiz** [comment]
This was fixed by #254. A known issue was added. https://docs.wpilib.org/en/stable/docs/yearly-overview/known-issues.html#visual-studio-code-extensions-fail-to-install-on-macos
